### ISSUE TITLE
PyJWT Version Requirement Conflict When Installing Superset from Scratch 

### ISSUE THREAD

**Issue by chriscardillo** [apache/superset#12849]
### Expected results

I expected that following the [instructions on installing Superset from scratch](https://superset.apache.org/docs/installation/installing-superset-from-scratch) would allow me to install Superset.

### Actual results

`pip install superset` does not install all of the required packages, because there is a dependency incompatibility issue in the package list.

#### Screenshots

`ERROR: flask-jwt-extended 3.25.0 has requirement PyJWT<2.0,>=1.6.4, but you'll have pyjwt 2.0.1 which is incompatible.`

#### How to reproduce the bug

While in your home directory...

```
mkdir superset
cd superset/
python3 -m venv venv
source venv/bin/activate
pip install apache-superset
```

The resulting error will be:

`ERROR: flask-jwt-extended 3.25.0 has requirement PyJWT<2.0,>=1.6.4, but you'll have pyjwt 2.0.1 which is incompatible.`

If we try to run `superset db upgrade` after that, here is what we see:

```
Traceback (most recent call last):
  File "/home/chris/work/superset/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/home/chris/work/superset/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/chris/work/superset/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 791, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (PyJWT 2.0.1 (/home/chris/work/superset/venv/lib/python3.8/site-packages), Requirement.parse('PyJWT<2.0,>=1.6.4'), {'Flask-JWT-Extended'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/chris/work/superset/venv/bin/superset", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/home/chris/work/superset/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3252, in <module>
    def _initialize_master_working_set():
  File "/home/chris/work/superset/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3235, in _call_aside
    f(*args, **kwargs)
  File "/home/chris/work/superset/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3264, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/home/chris/work/superset/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 585, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/home/chris/work/superset/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 598, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/home/chris/work/superset/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 791, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (PyJWT 2.0.1 (/home/chris/work/superset/venv/lib/python3.8/site-packages), Requirement.parse('PyJWT<2.0,>=1.6.4'), {'Flask-JWT-Extended'})
```

### Environment

- superset version: 1.0.0
- python version: 3.8.5
- node.js version: (I do not have Node installed)

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

NA

**Comment by amitmiran137** [comment]
this is resolved in 1.0.1
https://github.com/apache/superset/pull/12804

**Comment by mostafaghadimi** [comment]
I have had the same problem using Flask, it was resolved after downgrading it to a specific version. For more details you can see the whole problem and answer [here](https://stackoverflow.com/questions/67750044/how-to-resolve-versionconflict-error-in-flask-pyjwt-and-flask-jwt-extended).
